.github/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  rust:
+    uses: primait/shared-github-actions/.github/workflows/rust-ci.yml@rust-ci-v1
```

Comment:
this won't work for a public repo. You can't use the workflows defined in a private shared-github-actions repo in a public repo

Reply:
I felt there was a reason but I couldn't remember it, thanks :pray: